content/docs/instrumenting/exporters.md

```diff
@@ -111,6 +111,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Hadoop HDFS FSImage exporter](https://github.com/marcelmay/hadoop-hdfs-fsimage-exporter)
    * [Lustre exporter](https://github.com/HewlettPackard/lustre_exporter)
    * [ScaleIO exporter](https://github.com/syepes/sio2prom)
+   * [TSM exporter](https://github.com/treydock/tsm_exporter)
```

Comment:
I don't know what TSM is, spell it out.

Reply:
So should I call it the official product name that was a result of recent re-branding by IBM, which is `IBM Spectrum Protect` or the more common name used by people using the product that is `Tivoli Storage Manager`?  The product itself didn't change when named changed, was just a rebranding.

Reply:
What would a user looking for this call it? If both are plausible, both can be listed.

Reply:
I went ahead and listed both since the rename is somewhat recent and so people might be looking for either old or new name.

Reply:
> What would a user looking for this call it? If both are plausible, both can be listed.

Some input from a TSM user: We usually call it "TSM" despite knowing that it has been renamed. Filesystem paths still refer to the old name, while license stuff uses the new name. Users will recognize both, but in order to make it simpler to find such an exporter, it would make sense to list both names in my opinion.